docs/how/search-over-new-field.md

```diff
@@ -256,7 +256,7 @@ curl 'http://localhost:8080/corpUsers?action=ingest' -X POST -H 'X-RestLi-Protoc
 Once the ingestion is done, you can test your changes by issuing search queries. Here is an example query with response.
 
 ```sh
-curl "http://localhost:8080/corpUsers?q=search&input=ai200" -H 'X-RestLi-Protocol-Version: 2.0.0' -s | jq
+curl "http://localhost:8080/corpUsers?q=search&input=ai100" -H 'X-RestLi-Protocol-Version: 2.0.0' -s | jq
```

Comment:
The course name being ingested as data during ingestion is `AI100: Introduction to Artificial Intelligence`

Searching `ai200` gives empty results.